### PR TITLE
Created md for set_remote_params.py and basic ros param usage

### PR DESCRIPTION
**Pull request by steph-lheureux**
Created a document describing usage of the `SetRemoteParams` class from [`set_remote_params.py`](https://github.com/CabrilloRoboticsClub/cabrillo_rov_2023/blob/main/src/seahawk/seahawk_deck/set_remote_params.py) and ROS params.

### CODE REVIEW

**Review by mike-matera — APPROVED** [review]
Looks good!

**Review by tinymassi — APPROVED** [review]
While I don't understand parameters very well yet, this documentation has some pretty helpful information for helping those who are new to them! Well done!

File: doc/params.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# ROS Params
+[ROS parameters](https://docs.ros.org/en/humble/Concepts/Basic/About-Parameters.html) are a way of setting attributes of a node which can be accessed and modified by the ROS network
+
+## Usage
+### Client node
+*The client node is the node which makes a request to update another node's parameters*
+1. Import the `RemoteParamsClient` class to any node on the deck with the following syntax. The `SetRemoteParams` class from [`set_remote_params.py`](https://github.com/CabrilloRoboticsClub/cabrillo_rov_2023/blob/main/src/seahawk/seahawk_deck/set_remote_params.py) provides functionality for nodes to modify each other's [ROS parameters](https://docs.ros.org/en/humble/Concepts/Basic/About-Parameters.html) using [services](https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Services/Understanding-ROS2-Services.html). The `SetRemoteParams` class streamlines setting another node's parameters to a user-friendly abstraction to reduce redundancy creating cleaner code and increased ease-of-use.
+	```py
+	from .set_remote_params import RemoteParamsClient
```
Comment (liamgilligan) [inline doc/params.md:9]: Maybe generalize this to the ROV as well

File: doc/params.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
+# ROS Params
+[ROS parameters](https://docs.ros.org/en/humble/Concepts/Basic/About-Parameters.html) are a way of setting attributes of a node which can be accessed and modified by the ROS network
+
+## Usage
+### Client node
+*The client node is the node which makes a request to update another node's parameters*
+1. Import the `RemoteParamsClient` class to any node on the deck with the following syntax. The `SetRemoteParams` class from [`set_remote_params.py`](https://github.com/CabrilloRoboticsClub/cabrillo_rov_2023/blob/main/src/seahawk/seahawk_deck/set_remote_params.py) provides functionality for nodes to modify each other's [ROS parameters](https://docs.ros.org/en/humble/Concepts/Basic/About-Parameters.html) using [services](https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Services/Understanding-ROS2-Services.html). The `SetRemoteParams` class streamlines setting another node's parameters to a user-friendly abstraction to reduce redundancy creating cleaner code and increased ease-of-use.
```
Comment (liamgilligan) [inline doc/params.md]: Grammar mistake, should have a comma between redundancy and creating: `...to reduce redundancy, creating cleaner...`

**Review by liamgilligan — APPROVED** [review]
Looks good! Glad to have some more documentation!

**Pull request merged**